spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -164,7 +125,7 @@ public ProcessorConfigurer via(Class<?> processor) {
 
 	}
 
-	private abstract class AppConfigurer {
+	public abstract class AppConfigurer<T extends AppConfigurer<T>> {
```

Comment:
Is this meant to be just `AppConfigurer<T>` unless I miss something very basic :-)